src/lib/fetchVoiceAPI.ts

```diff
@@ -0,0 +1,47 @@
+interface res {
+  success: boolean;
+  host: string;
+  audioId: string;
+  audioStatusUrl: string;
+  wavDownloadUrl: string;
+  mp3DownloadUrl: string;
+}
+
+let txt: string = "";
+let speaker: number = 0;
+
+export class fetchVoiceAPI {
+  constructor(inputTxt: string, speakerId: number) {
+    txt = inputTxt;
+    speaker = speakerId;
+  }
```

Comment:
@yamada1023 
↓みたいに変数をクラス内にカプセル化できないですかね？👀

これまでのコードにもあったような気はするのですが、クラスの外に変数を宣言するのがどうも違和感がありまして…

↓あくまでイメージ
```suggestion
export class fetchVoiceAPI {
  private static txt = "";
  private static speaker = 0;

  constructor(inputTxt: string, speakerId: number) {
    fetchVoiceAPI.txt = inputTxt;
    fetchVoiceAPI.speaker = speakerId;
  }
}
```